_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,271 @@
+- **named from author**: the element is a [descendant][] in the [flat tree][] of an element that is [named from author][].
+
+**Note**: An example of an image ignored because of "named from author" is when it in a `button` element that uses `aria-label` for its accessible name.
+
+## Expectation
+
+Each test target is [purely decorative][].
+
+**note**: It is relatively common for an informative image such as an icon to be marked up as decorative, if the text alternative is adjacent to the image. This is a [conforming alternative version][] for the image. This fails the rule, but meets [conformance requirement 1 of WCAG 2.1](https://www.w3.org/TR/WCAG21/#cc1).
+
+## Assumptions
+
+- `svg` elements with a [semantic role][] of `graphics-document` and with an empty (`""`) [accessible name][] are ignored by assistive technologies tested for this rule. If some assistive technology does not ignore these elements, and that assistive technology is required for conformance, passing this rule does not ensure all decorative `svg` elements can be ignored, and the [success criterion 1.1.1 Non-text content][] may still not be satisfied. The same is true for `canvas` elements with no [semantic role][] and an empty (`""`) [accessible name][].
+
+- A web page with informative images without an [accessible name][] may conform to WCAG 2.1 Level A when the information provided by that image is available elsewhere, either on the web page itself or on a [conforming alternative version][] of the page. For example if an equivalent text is adjacent to the image, or if the text alternative is included in the [accessible name][] of a parent element.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [H67: Using null alt text and no title attribute on img elements for images that AT should ignore](https://www.w3.org/WAI/WCAG21/Techniques/html/H67.html)
+- [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This decorative `img` element is ignored by assistive technologies because of the `alt` attribute is empty (`""`).
+
+```html
+Happy new year!<br />
+<img src="/test-assets/shared/fireworks.jpg" alt="" />
+```
+
+#### Passed Example 2
+
+This decorative `img` element is ignored by assistive technologies because `aria-hidden` is set to `true`.
+
+```html
+Happy new year!<br />
+<img src="/test-assets/shared/fireworks.jpg" aria-hidden="true" alt="decorative" />
+```
+
+#### Passed Example 3
+
+This decorative `img` element is ignored by assistive technologies because it has an [explicit semantic role][] of `none`.
+
+```html
+Happy new year!<br />
+<img src="/test-assets/shared/fireworks.jpg" role="none" alt="ignore me" />
+```
+
+#### Passed Example 4
+
+This decorative `svg` element is ignored by assistive technologies because it has no attribute that would give it an [accessible name][].
```

Comment:
I guess this rule is not to check whether element have accessible name or not? We should rephrase this description to meet the expectation and also for similar examples. Something like that:
"This `svg` element is not included in the accessibility tree and is a decorative because it do not contain an attribute which provide alternative text to this element."